lms/djangoapps/instructor/apps.py

```diff
@@ -22,8 +21,8 @@ class InstructorConfig(AppConfig):
         PluginURLs.CONFIG: {
             ProjectType.LMS: {
                 PluginURLs.NAMESPACE: '',
-                PluginURLs.REGEX: f'^courses/{COURSE_ID_PATTERN}/instructor/api/',
-                PluginURLs.RELATIVE_PATH: 'views.api_urls',
+                PluginURLs.REGEX: '',
+                PluginURLs.RELATIVE_PATH: 'urls',
             }
```

Comment:
I made this change because I didn't want the new APIs to have a path like:
`/courses/<course_id>/instructor/api/v0/...'
The new APIs are at:
`/instructor/api/v0/courses/<course_id>/...`